src/runner/kafka.js

```diff
@@ -45,11 +46,12 @@ class KafkaRunner implements IRunner {
     }
   }
 
-  process() {
+  process(filterTopics: Array<string>) {
     const subscriptions = this.registry.getTopics();
+    const filtered = difference(subscriptions, filterTopics);
```

Comment:
hmm, does this mean the `filteredTopics` will need to be the same as the name generated with the environment OR will it filter by the task name